internal/runner/command.go

```diff
@@ -75,6 +75,8 @@ type commandConfig struct {
 	Script   string
 
 	Logger *zap.Logger
+
+	Winsize *pty.Winsize
```

Comment:
Nitpick: it's pty-related so I would group it with the fields starting with `Tty bool`.

Reply:
Oh yeah, that's fair!